persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -221,7 +221,15 @@ selectAllConversations:
 SELECT * FROM ConversationDetails WHERE type IS NOT 'CONNECTION_PENDING' ORDER BY last_modified_date DESC, name ASC;
 
 selectAllTeamProteusConversations:
-SELECT * FROM ConversationDetails WHERE type IS 'GROUP' AND protocol IS 'PROTEUS' AND teamId = ?;
+SELECT qualified_id FROM Conversation WHERE type IS 'GROUP' AND protocol IS 'PROTEUS' AND team_id = ?;
+
+selectAllTeamProteusConversationsReadyForMigration:
+SELECT
+qualified_id,
+(SELECT count(*) FROM Member WHERE conversation = qualified_id) AS memberCount,
+(SELECT count(*) FROM Member LEFT JOIN User ON User.qualified_id = Member.user WHERE Member.conversation = Conversation.qualified_id AND User.supported_protocols LIKE '%mls%') AS mlsCapableMemberCount
```

Comment:
I suggest to not use 'Like' command since it's heavier, and AFAIK we have only couple of known states, so please let's have them compared directly.

Reply:
What do you suggest? Listing all variations explicitly?

```
(User.supported_protocols = "MLS" OR 
 User.supported_protocols = "MLS,PROTEUS" OR 
 User.supported_protocols = "PROTEUS,MLS")
```

Reply:
Updated